roles/openshift_preflight/facts/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: set common variables
+  set_fact:
+    deployment_type: "{{ deployment_type | default('openshift-enterprise') }}"
```

Comment:
I agree.

@brenton gave an interesting suggestion which I think we could implement by checking variables defined in the std facts:

> <bleanhar> if we detect RHEL or Atomic Host we default to running the OCP checks
> <bleanhar> anything else we run the Origin checks
> <bleanhar> and then let it be overridden as needed for advanced cases

Do you think that is reasonable? Is there a standard way to check for RHEL or Atomic Host?